src/theme/index.ts

```diff
@@ -0,0 +1,89 @@
+export const colors = {
+    WHITE: '#FFFFFF',
+
+    GRAY: {
+        900: '#111827',
+        800: '#1F2937',
+        700: '#374151',
+        600: '#4B5563',
+        500: '#6B7280',
+        400: '#9CA3AF',
+        300: '#D1D5DB',
+        200: '#E5E7EB',
+        100: '#F3F4F6',
+        50: '#E5E7EB',
+    },
+
+    RED: {
+        900: '#771D1D',
+        800: '#9B1C1C',
+        700: '#C81E1E',
+        600: '#E02424',
+        500: '#F05252',
+        400: '#F98080',
+        300: '#F8B4B4',
+        200: '#FBD5D5',
+        100: '#FDE8E8',
+        50: '#FDF2F2',
+    },
+
+    GREEN: {
+        900: '#014737',
+        800: '#03543F',
+        700: '#046C4E',
+        600: '#057A55',
+        500: '#0E9F6E',
+        400: '#31C48D',
+        300: '#84E1BC',
+        200: '#BCF0DA',
+        100: '#DEF7EC',
+        50: '#F3FAF7',
+    },
+
+    BLUE: {
+        900: '#233876',
+        800: '#1E429F',
+        700: '#1A56DB',
+        600: '#1C64F2',
+        500: '#3F83F8',
+        400: '#76A9FA',
+        300: '#A4CAFE',
+        200: '#C3DDFD',
+        100: '#E1EFFE',
+        50: '#EBF5FF',
+    },
+
+    BLACK: {
+        200: '#00000040',
+        100: '#0000001A',
+        50: '#0000000F',
+    },
+};
+
+export const radius = {
+    'ROUNDED-FULL': '50%',
+    'ROUNDED-2XL': '16px',
+    'ROUNDED-XL': '10px',
+};
+
+export const borders = {
+    'GRAY-1': `solid 1px ${colors.GRAY[200]}`,
+    'BLACK-1': `solid 1px ${colors.BLACK[200]}`,
+};
+
+export const boxShadow = {
+    'SHADOW-SM': `0px 1px 3px 0px ${colors.BLACK[100]}, 0px 1px 2px 0px ${colors.BLACK[50]}`,
+    'SHADOW-MD': `0px 4px 4px 0px ${colors.BLACK[200]}`,
+};
+
+export const breakpoints = {
+    MOBILE: '320px',
+    TABLET: '768px',
+    SMALLDESKTOP: '1024px',
+    DESKTOP: '1440px',
+    LARGEDESKTOP: '1920px',
+};
```

Comment:
We won't be needing these many breakpoints

```suggestion
export const breakpoints = {
    MOBILE: '320px',
    TABLET: '768px',
    SMALL_DESKTOP: '1024px',
    DESKTOP: '1440px',
    LARGE_DESKTOP: '1920px',
};
```

Reply:
Just in case.